apps/app/src/index.tsx

```diff
@@ -22,7 +24,7 @@ datadogRum.init({
   site: 'datadoghq.com',
   service: pkg.name,
   env: process.env.REACT_APP_ENV_NAME,
-  version: pkg.version,
```

Comment:
nice, I guess we aren't updating the app version so this wouldn't have been helpful as it was 👍